bin/tigmint-make

```diff
@@ -161,6 +193,20 @@ $(draft).tigmint.arcs.fa: $(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsi
 $(draft).%.sortbx.bam: %.fq.gz $(draft).fa.bwt
 	$(gtime) bwa mem -t$t -pC $(draft).fa $< | samtools view -u -F4 | samtools sort -@$t -tBX -T$$(mktemp -u -t $@.XXXXXX) -o $@
 
+# Segment long reads from gzipped fasta file.
+$(reads).cut$(cut).fa.gz: $(lr_filename)
+ifeq ($(span), auto)
+	$(gzip) -dc $< | $(bin)/long-to-linked -l$(cut) -m$(minsize) -g$(gsize) -s | $(gzip) > $@
```

Comment:
I notice the default output for the `span` parameter is `tigmint-long.span.txt`? Might be good to set the `-o` parameter here so that it named for the ONT reads specifically -- not super important but just in the remote chance that there are multiple `tigmint-long` runs with different read sets in the same directory?